packages/home-spa/package.json

```diff
@@ -38,7 +38,7 @@
     "node-sass": "4.14.1",
     "sass-loader": "8.0.2",
     "style-loader": "1.2.1",
-    "terser-webpack-plugin": "2.3.6",
+    "terser-webpack-plugin": "3.0.1",
```

Comment:
@hybridx Can you test this new version? There seem to be some breaking changes related to `extractComments`.

Reply:
@deshmukhmayur These changes LGTM